models/articles.model.js

```diff
@@ -65,4 +69,33 @@ exports.insertCommentById = (article_id, username, body) => {
           return data.rows[0];
         });
     });
+}
+
+function incVotesById(article_id, inc_votes) {
+  return selectArticleById(article_id).then((data) => {
+    if (isNaN(inc_votes)) {
+      return Promise.reject({
+        status: 400,
+        message: "Cannot update. Invalid data.",
```

Comment:
You can be even more specific with this error message and really let the user know what is wrong.